app/assets/stylesheets/views/_brexit-landing-page.scss

```diff
@@ -34,6 +35,13 @@ $red: #E61E32;
   }
 }
 
+.landing-page__page_header {
```

Comment:
This should be `.landing-page__page-header` - but actually would it be possible to change it to something else? I was briefly confused by the previous class in this PR `.landing-page__header`. Maybe `.landing-page__title`?

Reply:
@andysellick discussed with @vanitabarrett we opted for `.landing-page__page-title` you think that would have been clearer or still cause the pause for thought?